onset_maproom/enactstozarr.py

```diff
@@ -43,9 +46,12 @@ def set_up_dims(xda):
     # and look into xESMF.
```

Comment:
could this be the culprit for the bands of empty data when zooming the Senegal map? I guess not because you are not doing regridding for Senegal, and you wrote the rechunked Senegal data after seeing the bands... correct?

Reply:
Correct on both counts.